stories/content.stories.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import Content from './../src/web/components/Content';
+
+// TODO remove html entities when they are implemented
```

Comment:
Not needed for here, storybooks only aim to provide feedback on UI. It's not really bad not to use the other components right there :).

I think you can simply remove this comment